payloads/external/edk2/Kconfig.dasharo

```diff
@@ -105,6 +105,17 @@ config EDK2_DASHARO_INTEL_ME_OPTIONS
 	help
 	  Enables the Intel ME configuration submenu in Dasharo System Features.
 
+if EDK2_DASHARO_INTEL_ME_OPTIONS
+
+config EDK2_DASHARO_INTEL_ME_CBNT_STATUS
+	bool "Enable CBnT status on the Intel ME configuration page"
+	default n
```

Comment:
`depend on INTEL_CBNT_SUPPORT && EDK2_DASHARO_INTEL_ME_OPTIONS` ? (then IF can be removed)

Reply:
https://github.com/Dasharo/coreboot/pull/691/commits/ba16b069a312dbba4de4260a08a0d00ef0dbe37a

No change to defconfigs necessary, seems to work correctly.